cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -160,6 +161,13 @@ def _write_chunk(request, courselike_key):
             # This shouldn't happen, even if different instances are handling
             # the same session, but it's always better to catch errors earlier.
             if size < int(content_range['start']):
+                log.warning(
+                    "Error in Uploading course %s with file path is %s also with range %s on server %s",
+                    courselike_key,
+                    temp_filepath if temp_filepath else 'EMPTY',
```

Comment:
is there a case where `temp_filepath` can not be initialized or its fasly?

Reply:
@awaisdar001 I am not pretty sure about in which case where `temp_filepath` would be empty that's why I just added this check.

Reply:
`temp_filepath` is just a path object. you do something like `temp_filepath.exists()` or maybe something similar which I am not sure about the syntax.

Reply:
okay @awaisdar001 I will change it. :)